### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by dwillmer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,19 +11,15 @@ var phosphide = require('phosphide');
...
+  require('phosphide/lib/commandregistry/plugin'),
+  require('phosphide/lib/commandpalette/plugin'),
+  require('red'),
+  require('blue'),
+  require('green'),
+  require('yellow'),
+  require('editor')
+]).then(function() {
+  console.log('loading finished');
+});
```
Comment (sccolbert) [inline index.js]: why these changes?

File: blue/tsconfig.json [hunk 2]
```diff
@@ -3,7 +3,6 @@
     "noImplicitAny": true,
     "noEmitOnError": true,
     "module": "commonjs",
-    "moduleResolution": "node",
```
Comment (sccolbert) [inline blue/tsconfig.json]: this should not be removed

File: red/tsconfig.json [hunk 3]
```diff
@@ -3,7 +3,6 @@
     "noImplicitAny": true,
     "noEmitOnError": true,
     "module": "commonjs",
-    "moduleResolution": "node",
```
Comment (sccolbert) [inline red/tsconfig.json]: this should not be removed

File: index.js (hunk 1 above)
Comment (dwillmer) [inline index.js]: was just updating it to be in line with the example in phosphide

Comment (sccolbert) [inline index.js]: I think someone put that there in the switch to webpack. The phosphide example is still using StealJS.
